packages/theme/src/theme.ts

```diff
@@ -65,41 +59,71 @@ export class Theme extends HTMLElement {
     }
 
     get color(): Color {
-        const color = this.getAttribute('color');
-        if (!color) return DefaultColor;
-
-        const colorFragment = Theme.themeFragments.get(color);
-        if (colorFragment && colorFragment.kind === 'color') {
-            return color as Color;
-        }
-        return DefaultColor;
+        return this.getFragementNameByKind('color') as Color;
     }
 
     set color(newValue: Color) {
         this.setAttribute('color', newValue);
     }
 
     get scale(): Scale {
-        const scale = this.getAttribute('scale');
-        if (!scale) return DefaultScale;
-
-        const scaleFragment = Theme.themeFragments.get(scale);
-        if (scaleFragment && scaleFragment.kind === 'scale') {
-            return scale as Scale;
-        }
-        return DefaultScale;
+        return this.getFragementNameByKind('scale') as Scale;
     }
 
     set scale(newValue: Scale) {
         this.setAttribute('scale', newValue);
     }
 
+    private getFragementNameByKind(kind: string): string {
+        const kindFragments = Theme.themeFragmentsByKind.get(
+            kind
+        ) as FragmentMap;
+        /* istanbul ignore if */
+        if (!kindFragments) {
+            throw new Error(`Unknown theme fragment kind '${kind}'`);
+            /* istanbul ignore if */
+        } else if (kindFragments.size === 0) {
+            throw new Error(`No theme fragments of kind '${kind}'`);
```

Comment:
This is probably not needed, as the only place we add theme fragments right now create the kindMap and adds the fragments, but there were `errors` thrown before and I wanted to join the party, even if I'm not good at knowing when to throw what. As this would only be encountered when addressing this element imperatively after having queried for it in the DOM, maybe we can drop it?